platform/client.go

```diff
@@ -0,0 +1,128 @@
+package platform
+
+import (
+	"fmt"
+
+	"github.com/Peripli/service-broker-proxy/pkg/platform"
+	"github.com/cloudfoundry-community/go-cfclient"
+	"github.com/sirupsen/logrus"
+)
+
+type PlatformClient struct {
+	cfClient *cfclient.Client
+	reg      *RegistrationDetails
+}
+
+var _ platform.Client = &PlatformClient{}
```

Comment:
It took me some time to understand where this `platform.Client` comes from. May be use some alias for the imported platform. Currently I don't have better idea for name :(